### PR TITLE
Fix CI build on windows

### PR DESCRIPTION
**Pull request by cbley-da**
Prior to this change the installation of msys2 would fail because packages fail to validate their signature:

```
:: File /var/cache/pacman/pkg/perl-HTTP-Cookies-6.10-1-any.pkg.tar.zst is corrupted (invalid or corrupted package (PGP s
ignature)).
Do you want to delete it? [Y/n]
error: perl-Net-SSLeay: signature from "David Macek <david.macek.0@gmail.com>" is unknown trust
:: File /var/cache/pacman/pkg/perl-Net-SSLeay-1.90-1-x86_64.pkg.tar.zst is corrupted (invalid or corrupted package (PGP
signature)).
Do you want to delete it? [Y/n]
error: failed to commit transaction (invalid or corrupted package)
```

### CODE REVIEW

File: dev-env/windows/manifests/msys2.json [hunk 1]
```diff
@@ -1,108 +1,90 @@
...
-    },
-    "uninstaller": {
-        "script": [
-            "rm -r -Force $dir"
-        ]
+    "license": "GPL-2.0-only|BSD-3-Clause",
+    "notes": "Please run 'msys2' now for the MSYS2 setup to complete!",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/msys2/msys2-installer/releases/download/2022-01-28/msys2-base-x86_64-20220128.tar.xz",
```
Comment (cocreature) [inline dev-env/windows/manifests/msys2.json:10]: why can we drop all the other packages?

**Comment by garyverhaegen-da** [comment]
Thanks for looking into this! It looks like this got us past the Scoop install step. I'll go have dinner now (haven't eaten yet today due to transit) but will take a look later at the final result of the build, and I'll try to get it sorted by tomorrow morning.

File: dev-env/windows/manifests/msys2.json (hunk 1 above)
Comment (garyverhaegen-da) [reply dev-env/windows/manifests/msys2.json:10]: So I have an answer to that: we can't. Scoop now works but we fail at the very first Bazel task on a missing `patch` executable.

**Review by garyverhaegen-da — APPROVED** [review]

File: dev-env/windows/manifests/msys2.json (hunk 1 above)
Comment (cbley-da) [reply dev-env/windows/manifests/msys2.json]: Thank you Gary, for sorting this out -- I did not really understand the purpose of the entries in the manifest file and oversimplified way too much.

**Pull request merged**